tests/cpp/test_multidevice_matmul.cpp

```diff
@@ -62,6 +62,10 @@ TEST_F(DistributedMatmulTest, MulSum_LayoutTN_NoComms) {
   // MmaLayout::TN A(T), B(N), C(T)
   // A and C are sharded on dimension M
   // Tests local matmul with no communication
+  auto dev_prop = at::cuda::getCurrentDeviceProperties();
+  if (dev_prop->major > 9) {
+    GTEST_SKIP() << "Require Ampere or Hopper for generated matmul";
```

Comment:
Note that you could use one of these macros if it helps: https://github.com/NVIDIA/Fuser/blob/f097e58ea1ae9d2c70115ce6dfabc7b27763fcb3/tests/cpp/utils.h#L510

Reply:
Thanks!